freqtrade/optimize/backtesting.py

```diff
@@ -136,6 +137,10 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self.config['startup_candle_count'] = self.required_startup
         self.exchange.validate_required_startup_candles(self.required_startup, self.timeframe)
 
+        self.enable_backtest_signal_candle_export = False
+        if self.config.get('enable_backtest_signal_candle_export', None) is not None:
+            self.enable_backtest_signal_candle_export = bool(self.config.get('enable_backtest_signal_candle_export'))
```

Comment:
This can be greatly simplified ... 

```suggestion
        self.enable_backtest_signal_candle_export = self.config.get('enable_backtest_signal_candle_export', False)
```

If you want to ensure the parameter is a bool, please do so using the json validator in `constants.py`.

I would also want to ensure this is disabled in hyperopt - as this WILL have an impact on performance eventually.

Reply:
Addressed in latest commits